File: Algorithm/Qlearning_Python/env/reset_mObs.py
def reset_mObs(initialMovingObs, hMObs):
    countObs = initialMovingObs.shape[0]
    if countObs != 0:
        hMObs = []
        movingObs = initialMovingObs
        for i in range(0, countObs):
            startObs = movingObs[i, 0:2]
            # radObs = movingObs[i, 3]
            hMObs.append(startObs)
    else:
        movingObs = [];
        hMObs = [];
    return [movingObs, hMObs]

File: Algorithm/QLearning_Matlab/env/reset_mObs.m
function [movingObs, hMObs] = reset_mObs(initialMovingObs, hMObs)
%Update position of moving obstacle after a interval dt
[countObs, ~] = size(initialMovingObs);
if countObs ~= 0
    delete(hMObs);
    hMObs = [];
    movingObs = initialMovingObs;
    for i=1:size(countObs, 1)
        startObs = movingObs(i,1:2);
        radObs = movingObs(i,4);
        hObs = viscircles(startObs, radObs, 'LineWidth', 1, 'Color', 'yellow');
        hMObs = [hMObs hObs];
    end
else
    movingObs = [];
    hMObs = [];
end
